File: URI Solutions/1020.cpp
#include <iostream>

using namespace std;

int main(int argc, char *argv[])
{
    int t,y,m,d;
    while(cin>>t)
    {
        y = t / 365;
        m = (t - y*365) / 30;
        d = t -(y*365 + m *30);

        cout<<y<<" ano(s)\n"<<m<<" mes(es)\n"<<d<<" dia(s)\n";
    }
    
    return 0;
}

File: URI Solutions/1020.cs
using System;


class URI
{


    static void Main(string[] args)
    {
        int t, y, m, d;
        t = Convert.ToInt32(Console.ReadLine());

        y = t / 365;
        m = (t - y * 365) / 30;
        d = t - (y * 365 + m * 30);

        Console.WriteLine(y + " ano(s)\n" + m + " mes(es)\n" + d+" dia(s)");
    }
}
